test/jdk/java/nio/channels/Channels/TransferTo.java

```diff
@@ -139,16 +139,20 @@ public void testMoreThanTwoGB() throws IOException {
         Path targetFile = Files.createTempFile(null, null);
 
         // writing 3 GB of random bytes into source file
-        for (int i = 0; i < 3 * 1024; i++)
-            Files.write(sourceFile, createRandomBytes(1024 * 1024, 0), StandardOpenOption.APPEND);
+        int NUM_WRITES = 3 * 1024;
+        int BYTES_PER_WRITE = 1024 * 1024;
```

Comment:
We typically do not capitalize local variables, we  capitalize constants(Brian was suggesting the use of constants).

Using constants adds consistency with their use elsewhere.

If you choose to use a local variable, the variable name should not be capitalized

Reply:
Done in https://github.com/openjdk/jdk/pull/6093/commits/126e4877361ac7e3ab82101fee78d7d50c5733ac.